tests/ztest/base/src/main.c

```diff
@@ -35,6 +35,12 @@ ZTEST(framework_tests, test_assert_mem_equal)
 	zassert_mem_equal(actual, expected, sizeof(expected), NULL);
 }
 
+ZTEST(framework_tests, test_skip_config)
+{
+	Z_TEST_SKIP_IFDEF(CONFIG_BUGxxxxx);
+	zassert_true(false, NULL);
```

Comment:
actually, why not use ztest_test_skip() here, this way the test will be correctly counted as skipped.